encoder/vaapiencoder_base.cpp

```diff
@@ -57,6 +57,8 @@ VaapiEncoderBase::VaapiEncoderBase():
     m_videoParamCommon.rcParams.windowSize = 500;
     m_videoParamCommon.rcParams.disableBitsStuffing = 1;
     m_videoParamCommon.leastInputCount = 0;
+    m_videoParamCommon.rcParams.diffIPQP = 2;
```

Comment:
What is the 2 and 4 come from? Will it introduce quality problem to default encoder param?